src/app/(app)/missing-shoes/Components/MissingShoesFormBuilder.tsx

```diff
@@ -0,0 +1,261 @@
+"use client";
+
+import React from "react";
+import FormWrapper from "@/common/FormWrapper";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@/components/ui/form";
+import {
+  Select,
+  SelectContent,
+  SelectItem,
+  SelectTrigger,
+  SelectValue,
+} from "@/components/ui/select";
+import { FormTitle } from "@/common/FormTitle/FormTitle";
+import { Input } from "@/components/ui/input";
+import { useMask } from "@react-input/mask";
+import { DateTimePicker } from "@/common/DateTimePicker/DateTimePicker";
+import { CNIC_VALIDATOR_REGEX, PHONENUMBER_VALIDATOR_REGEX } from "@/lib/regex";
+import { ShoesTyes } from "@/constant/constant";
+
+export const missingShoesSchema = z.object({
+  color: z.string().min(1, {
+    message: "Color must be selected.",
+  }),
+  type: z.enum([
+    ShoesTyes.Boots,
+    ShoesTyes.DressShoes,
+    ShoesTyes.Sandals,
+    ShoesTyes.Sneakers,
+    ShoesTyes.Formal,
+    ShoesTyes.Other,
+  ]),
+  size: z.string().refine(
+    (val) => {
+      const sizeNumber = parseFloat(val);
+      return !isNaN(sizeNumber) && sizeNumber >= 1 && sizeNumber <= 50;
+    },
+    {
+      message: "Size must be a number between 1 and 50.",
+    }
+  ),
+  time: z.date().refine((val) => val <= new Date(), {
```

Comment:
can we use new Date().getTime instead of today complete date ?

Reply:
Fixed